signal_scanner_bot/messages.py

```diff
@@ -81,16 +79,7 @@ def process_signal_message(blob: Dict, api: API) -> None:
 
     # Check if twitter-to-signal should be on/off
     condensed = _condense_command(message)
-    if condensed == START_LISTENING:
-        notice = "==Auto Scanning Activated=="
-        listening = True
-    elif condensed == STOP_LISTENING:
-        notice = "==Auto Scanning Deactivated=="
-        listening = False
-    else:
-        return
-    log.info(notice)
-    env.STATE.LISTENING = listening
+    notice = str(env.STATE.update_listening_status(condensed))
```

Comment:
I think this isn't necessary anymore since the update function should always return a string?
```suggestion
    notice = env.STATE.update_listening_status(condensed)
```